### ISSUE TITLE
Why i always gets a DeprecationWarning?

### ISSUE THREAD

**Issue by SherlockGuo** [adamgruber/mochawesome#107]
i always get this warning:
`DeprecationWarning: Calling an asynchronous function without callback is deprecated.`
i trace it an it's stack is:  
```
(node:73406) DeprecationWarning: Calling an asynchronous function without callback is deprecated.
    at makeCallback (fs.js:103:12)
    at Object.fs.close (fs.js:521:20)
    at saveToFile (/Users/Sherlock/repositories/test_data/node_modules/mochawesome/lib/reportGenerator.js:82:8)
    at Runner.<anonymous> (/Users/Sherlock/repositories/test_data/node_modules/mochawesome/lib/mochawesome.js:111:9)
    at emitNone (events.js:91:20)
    at Runner.emit (events.js:185:7)
    at /Users/Sherlock/repositories/test_data/node_modules/mocha/lib/runner.js:813:12
    at /Users/Sherlock/repositories/test_data/node_modules/mocha/lib/runner.js:661:9
    at next (/Users/Sherlock/repositories/test_data/node_modules/mocha/lib/runner.js:295:14)
    at Immediate.<anonymous> (/Users/Sherlock/repositories/test_data/node_modules/mocha/lib/runner.js:339:5)
```

How can i solve this warning? It's annoying.

**Comment by adamgruber** [comment]
Haven't seen that warning. Can you provide some more info or some example code where this is happening?

**Comment by adamgruber** [comment]
So, this warning was added to node v7 and its coming from the `reportGenerator.js` file that calls `fs.close`. Should be an easy fix just changing it to `fs.closeSync`. I'll try to get this done and cut a release in the next day or two. Maybe sooner.

Have you considered upgrading to mochawesome v2?

**Comment by adamgruber** [comment]
Just released 1.5.5. Should fix things.
